src/plugins/intel_cpu/src/transformations/snippets/aarch64/pass/snippets_mark_skipped.cpp

```diff
@@ -241,6 +241,9 @@ bool SnippetsMarkSkipped::run_on_model(const std::shared_ptr<ov::Model>& m) {
         if (is_skipped_op(node)) {
             continue;
         }
+        if (ov::is_type<const ov::op::v0::FakeQuantize>(node)) {
+            SetSnippetsNodeType(node, snippets::pass::SnippetsNodeType::SkippedByPlugin);
+        }
```

Comment:
The right place to disable tokenization for some ops is Transformation pipeline. So you need to remove FQ from that list: https://github.com/openvinotoolkit/openvino/blob/779d459be41c1d0c91a44f77b946e324c82de1d0/src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp#L1122C78-L1122C90

Reply:
Fixed